config/set/php80.php

```diff
@@ -100,4 +100,10 @@
             new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'l', 'lt'),
             new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'n', 'ne'),
         ]);
+
+    $rectorConfig
+        ->ruleWithConfiguration(FunctionArgumentDefaultValueReplacerRector::class, [
```

Comment:
This can be merged with existing `ruleWithConfiguration(FunctionArgumentDefaultValueReplacerRector::clas` above it